datamade/committee-oversight
# Add committee metadata

## Overview

Closes #101. As described in the issue, this PR adds `chair` and `hide_rating` fields to the `CommitteeDetailPage` model for use elsewhere on the site. It also adds corresponding properties to the `CommitteeOrganization` model, so that this data is more easily accessible on the landing page.

## Testing Instructions

 * If you haven't already, follow the setup instructions in #99 
* Run migrations: `docker-compose exec app python manage.py migrate`
* Look at the landing page (http://localhost:8000/). There should be no chairs listed, and each committee should have a rating.
* Go into the Wagtail admin and create a new committee page. Give it a chair and select "hide rating"
* Go back to the landing page and make sure that the chair name shows up and the rating is "N/A"
